### PR TITLE
Fixes for Elixir 1.15.0

### PR DESCRIPTION
**Pull request by josevalim**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -19,7 +19,7 @@ defmodule Hex.MixProject do
 
   def application do
     [
-      extra_applications: [:ssl, :inets, :logger],
+      extra_applications: [:ssh, :ssl, :inets, :logger],
```
Comment (josevalim) [inline mix.exs:22]: Maybe test only?

Comment (ericmj) [reply mix.exs:22]: We use it to display the public key fingerprint for these tasks:
```
mix hex.repo show NAME
mix hex.repo list
```

**Comment by michaelklishin** [comment]
Not complaining but FYI: Hex adding a dependency on the `ssh` app broke RabbitMQ Debian package builds because `ssh` was not listed as a build-time (or any other) dependency 😅

**Comment by josevalim** [comment]
Ah, that's a good point. People relying on apt-get default Erlang distribution now have an additional dependency to handle. @ericmj what do you think about removing `:ssh` as a dependency and doing:

    function_exported?(Mix, :ensure_application!, 1) && Mix.ensure_application!(:ssh)

Inside the `hex.repo` task?

**Comment by ericmj** [comment]
A fix has been pushed. A new release will be out soon.

**Pull request merged**